package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@across-protocol/sdk-v2",
   "author": "UMA Team",
-  "version": "0.4.8",
+  "version": "0.5.0-beta.16",
```

Should this leave beta?
```suggestion
  "version": "0.5.0",
```

---

should we until we confirm stability?

---

I think it may be a good idea to drop this out of beta and use the minor versions to perform hot-fixes